src/Microsoft.Health.Fhir.Shared.Core/Features/Validation/ProfileValidator.cs

```diff
@@ -38,12 +39,20 @@ public ProfileValidator(IProvideProfilesForValidation profilesResolver, IOptions
 
         private Validator GetValidator()
         {
+            var expanderSettings = new ValueSetExpanderSettings
+            {
+                MaxExpansionSize = 20000, // Set your desired max expansion size here
```

Comment:
Where did we get the 20,000 from? Was that the default value for Firely until we upgraded?

Reply:
The current valueset expansion that is being used is ~17,000.  The 20,000 give a bit of a buffer, but I don't want much larger than necessary.